packages/mobile/e2e/src/usecases/Send.js

```diff
@@ -61,7 +61,7 @@ export default Send = () => {
     // Look for the latest transaction and assert
     await waitFor(element(by.text(`${randomContent}`)))
       .toBeVisible()
-      .withTimeout(45000)
+      .withTimeout(90000)
```

Comment:
was this failing? or why do we need more time?

Reply:
The test was failing in the specified timeout in Circle Ci. I added additional allowed timeout and it started passing. This timeout will take up to 90 seconds; however, if the text is visible before during that time it'll pass and continue.